keychain/src/extkey.rs

```diff
@@ -75,36 +75,25 @@ impl fmt::Display for Fingerprint {
 	}
 }
 
-#[derive(Serialize, Deserialize, Clone)]
-pub struct Identifier([u8; 20]);
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
+pub struct Identifier(String);
```

Comment:
I did this so we can easily serialize/deserialize identifiers in and out of the wallet.dat file.
I'm not totally happy with this but think it is ok for now.